roles/openshift_facts/molecule.yml

```diff
@@ -0,0 +1,20 @@
+---
+ansible:
+  playbook: tests/playbook.yml
+  group_vars:
+    test_group:
+    - deployment_type: origin
+dependency:
+  name: galaxy
+driver:
+  name: docker
+docker:
+  containers:
+  - name: test_host
```

Comment:
:+1: sounds good, as I get this futher along I will add a common openshift-ansible related prefix.